clang/lib/Driver/ToolChains/Haiku.cpp

```diff
@@ -281,3 +286,11 @@ void Haiku::addLibCxxIncludePaths(const llvm::opt::ArgList &DriverArgs,
 Tool *Haiku::buildLinker() const { return new tools::haiku::Linker(*this); }
 
 bool Haiku::HasNativeLLVMSupport() const { return true; }
+
+SanitizerMask Haiku::getSupportedSanitizers() const {
+  SanitizerMask Res = ToolChain::getSupportedSanitizers();
+
+  Res |= SanitizerKind::Address;
```

Comment:
Do you have a feasibility plan ? Otherwise, fuzzer for example might be an easier promise.

Reply:
There is a patch set in the haikuports recipe I am going to upstream.

Reply:
ah alright then..